templates/index.html

```
@@ -0,0 +1,89 @@
+    <title>Smart Notes MVP</title>
+
+    <link rel="stylesheet" href="{{ url_for('static', filename='style.css') }}">
+</head>
+
+<body>
+
+<div class="container">
+
+    <!-- HEADER -->
+    <div class="header">
+        <img src="{{ url_for('static', filename='images/logo-icon.svg') }}" class="icon">
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)